app/models/goal.py

```diff
@@ -1,6 +1,30 @@
+from re import T
```

Comment:
You're the second person I've seen import this but I'm not sure what effect it actually has (it doesn't seem explicitly referenced).

I'm curious, why the import?